src/Components/NavBar/Menus/NavBarNotifications.tsx

```diff
@@ -28,7 +28,7 @@ export const NavBarNotifications: React.FC<NavBarNotificationsProps> = props =>
       onMouseEnter={lockScroll}
       onMouseLeave={unlockScroll}
     >
-      <Notifications mode="dropdown" paginationType="infinite" {...props} />
+      <Notifications mode="dropdown" {...props} />
```

Comment:
We always do infinite scroll pagination and don't show a button anymore, so this prop can be removed.